src/Http/Client.php

```diff
@@ -179,7 +179,7 @@ public function __construct($config = [])
     /**
      * Get the cookies stored in the Client.
      *
-     * @return \Cake\Http\Client\CookieCollection
+     * @return CookieCollection
```

Comment:
why?

Reply:
Cause $this->_cookies is a object of Cake\Http\Cookie\CookieCollection. But it was return the object of \Cake\Http\Client\CookieCollection

Reply:
But please fix it to the FQCN as it was before, just the right one.

Reply:
Then you need to use the correct FQCN not just `CookieCollection`.

Reply:
OK, then I am reverting this changes.

Reply:
Just fix it to the expected `\Cake\Http\Cookie\CookieCollection` then.

Reply:
Just want to ensure... After fixing this I need to send a new pull request, right?

Reply:
No, you can add a 2nd commit on top of this one.